api/v1/swagger.yaml

```diff
@@ -3466,7 +3466,7 @@ paths:
                 minLength: 7
                 maxLength: 4096
                 description: The query to append to GET subjects. Only "absolutePath", "name", "tags" are allowed in subjectQuery.
-              collectors:
+              possibleCollectors:
```

Comment:
Need to make sure there are no references to the old attribute names anywhere in the other refocus-collector-related repos!

Reply:
is there any way to tie that to this pr? or should I just make a pr in refocus-collector and we merge them at the same time?